### ISSUE TITLE
Review Nicolas

### ISSUE THREAD

**Issue by nriss** [arkhn/ops-tools#3]
# 06.08.2021
So cool to use it! Some (minor) comments:

- The ip is not correct there:
![image](https://user-images.githubusercontent.com/48218773/128522722-68db5d60-7145-4d24-8891-9c00aafbb0d2.png)

- connection description : for me, it is not really intuitive, can you add (click on a line) on the title ?
![image](https://user-images.githubusercontent.com/48218773/128524012-33dae983-22ac-401e-9aba-c2f8aafc2564.png)

- Table preview: add a scroll item for tables with lots of column
![image](https://user-images.githubusercontent.com/48218773/128524455-4f26a1be-2fe0-462a-923b-e63fb5d5b908.png)

**Comment by elsiehoffet-94** [comment]
Thanks for your input! What's wrong with this IP?

**Comment by nriss** [comment]
0.0.0.0 is not the good ip, I have to use the server ip to access to the jupyter notebook.
@Jasopaum I think you have a better comprehension of the problem ?

**Comment by elsiehoffet-94** [comment]
I think this IP should stay localhost. To access this dashboard, you will need to use the IP of the server which hosts your jupyter notebook. 
So if you run this on your server you can visit localhost and see your dashboard. 
If you run this notebook on a jupyter instance deployed on another server, you should visit the server's IP to access it. (And make sure the port specified in the notebook is exposed)

**Comment by nriss** [comment]
> I think this IP should stay localhost. To access this dashboard, you will need to use the IP of the server which hosts your jupyter notebook.
> So if you run this on your server you can visit localhost and see your dashboard.
> If you run this notebook on a jupyter instance deployed on another server, you should visit the server's IP to access it. (And make sure the port specified in the notebook is exposed)

Yep I know, I just wanted to say that this link is not resolved if we are outside of the server and it could be confusing
